### ISSUE TITLE
bug/search-page: search results count message is confusing

### ISSUE THREAD

**Issue by StrawberryScot** [dylanlittle/team-moshpit-project#180]
### Severity

Medium

### Suggested Priority

Medium

### Observed Behaviour

The location of `Found 1 result(s)` can be confusing, as it can look like 1 User (a user that isn't displayed) has been found

<img width="573" height="503" alt="Image" src="https://github.com/user-attachments/assets/8283e8bf-9973-4fb7-9ad4-d3a5e194f113" />

### Expected Behaviour

`Found 1 result(s)` should be either be clearly separate from the `Users` section, by using borders for the `Artists` and `Users` containers

or

`Found 1 result(s)` should be replaced with `Found 1 artist(s)` and `Found 0 user(s)`

### Reproduction Steps

1. Navigate to `http://127.0.0.1:8080/search`
2. Type `mag` into either search bar
